### PR TITLE
Extended core support by implementing CLUSTER NO-TOUCH, CLIENT SETINFO command

### LINKED ISSUE #1237: Add support for new redis command CLIENT SETINFO
7.2.0 adds support for the new redis command CLIENT SETINFO. We need to add support, documented [here](https://redis.io/commands/CLIENT-SETINFO).

### LINKED ISSUE #1240: Add support for new redis command CLIENT NO-TOUCH
7.2.0 adds support for the new redis command CLIENT NO-TOUCH. We need to add support, documented [here](https://redis.io/commands/CLIENT-NO-TOUCH).

### PR DESCRIPTION
**Pull request by vladvildanov**
Closes #1240 
Closes #1237 

@shacharPash

### CODE REVIEW

File: tests/Predis/Command/Redis/ACL_Test.php [hunk 1]
```diff
@@ -89,7 +89,7 @@ public function testSetUserCreatesACLUser(): void
     /**
      * @group connected
      * @return void
-     * @requiresRedisVersion >= 7.0.0
+     * @requiresRedisVersion == 7.0.0
```
Comment (tillkruss) [inline tests/Predis/Command/Redis/ACL_Test.php:92]: Let's use `<=` and `>=` if you want to isolate tests. With the current notation it won't run reliably.

**Review by tillkruss — CHANGES_REQUESTED** [review]

File: tests/Predis/Command/Redis/ACL_Test.php (hunk 1 above)
Comment (vladvildanov) [reply tests/Predis/Command/Redis/ACL_Test.php:92]: Then we need to add a support for this, since it's doesn't accept intervals

Comment (vladvildanov) [reply tests/Predis/Command/Redis/ACL_Test.php]: Done

**Review by tillkruss — CHANGES_REQUESTED** [review]
This is too many unrelated changes for me to review and keep track of.

**Comment by vladvildanov** [comment]
@tillkruss This PR doesn't belong to 2.x, I'll keep it as draft, so don't worry about it 👌

**Pull request closed without merging**